<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvanceSettings</class>
 <widget class="QMainWindow" name="AdvanceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txtLinesToRead">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>19</y>
      <width>739</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Number of lines to read from OCR data and compare to category, leave blank to read all</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>19</y>
      <width>223</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lines to read</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>223</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy Percentage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAccPercentage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>739</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>How accurate the OCR data need to be in compare to the category, decides accuracy of the result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirmAdvanceSetting">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
